docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  nixos-homepage:
+    build:
+      context: .
+    environment:
+      HOST: "0.0.0.0"
+    ports:
+      - 8000:8000
+    volumes:
+      - .:/app
+    command:
+      - nix-shell
```

Comment:
For command rather use `nix develop --command serve` to be also asked to use the binary cache to speed up the build.

Reply:
done